cardinal/system/system_tokeniser.go

```diff
@@ -100,58 +107,98 @@ func (ts *TokeniserSystem) setupVrbAct() {
 	ts.responseLookup[enums.ActionTypeBreak] = []enums.ActionType{enums.ActionTypeBreak}
 }
 
+// setupGrammar initializes the grammar response tookup table with predifined grammar
+func (ts *TokeniserSystem) setupGrammar() {
+	ts.grammarLookup["THE"] = enums.GrammarTypeDefinitionArticle
+	ts.grammarLookup["TO"] = enums.GrammarTypePreposition
+	ts.grammarLookup["AT"] = enums.GrammarTypePrepo
+	ts.grammarLookup["Around"] = enums.GrammarTypeAdverb
+}
+
 // FishTokens processes the tokenized command and returns VerbData
 func (ts *TokeniserSystem) FishTokens(tokens []string) component.VerbData {
 	var data component.VerbData // Initialize VerbData to store the result
-	var err uint8               // Error code variable
+	var err uint8 = 0           // Error code variable
```

Comment:
see Go zero values https://go.dev/tour/basics/12 we don't need to add an explicit value the compiler does this for us